### PR TITLE
Add instructions for locally building

### LINKED ISSUE #8: Local build of tutorial
@bouweandela, is there any way that we can make a local build of the tutorial to test it before making a PR?

If there is, can we include it in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by bouweandela**
Closes #8

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -126,6 +124,22 @@ your clone of the originating `gh-pages` branch.
 Lastly, published copies of all the lessons are available in the `gh-pages` branch of the originating
 repository for reference while revising.
 
+## Previewing your changes locally
+
+Extensive instructions for building and viewing the pages locally can be found [here](https://carpentries.github.io/lesson-example/setup.html). To get started quickly, run
+```bash
+sudo apt install ruby-dev ruby-bundler
+```
```
Comment (ledm) [inline CONTRIBUTING.md]: Maybe we need to widen this to include other sources? I'm on fedora, for instance. 

```suggestion
```bash
sudo apt install ruby-dev ruby-bundler
```
or 
```bash
 sudo dnf install rubygem-bundler ruby-devel
```
```

**Comment by ledm** [comment]
Just confirming that it works before I can approve, but I need to update ruby on my machine first.

File: CONTRIBUTING.md (hunk 1 above)
Comment (bouweandela) [reply CONTRIBUTING.md]: I guess adding instructions for more operating systems is great. Could you add what's needed for fedora?

Comment (ledm) [reply CONTRIBUTING.md]: Perhaps instead we could make a `environment.yml` conda for the tutorial?

Comment (bouweandela) [reply CONTRIBUTING.md]: I'm not sure how much ruby packages there are on conda. If you want something os independent, we could mention the `make docker-serve` command? That also works fine, but then you have to install docker first.

**Pull request merged**